CalorieTracker/CalorieTracker/Model/CalorieCount+Convenience.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CalorieCount.swift
+//  CalorieTracker
+//
+//  Created by Kat Milton on 8/23/19.
+//  Copyright © 2019 Kat Milton. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+extension CalorieCount {
+    
+    convenience init(intakeNumber: Double, date: Date? = Date(), context: NSManagedObjectContext = CoreDataStack.shared.mainContext) {
```

Comment:
Given that you might not care about the result of this call, it would be nice to leverage the `@discardableResult` keyword so that you don't have to have the `let calorieCount = CalorieCount(intakeNumber: intakeAmount))` in the `CalorieCountController` and can instead just do CalorieCount(intakeNumber: intakeAmount)).

```swift
@discardableResult convenience init(intakeNumber: Double, date: Date? = Date(), context: NSManagedObjectContext = CoreDataStack.shared.mainContext) {
        
        self.init(context: context)

          self.intakeNumber = intakeNumber
         self.date = date
     }
```